### PR TITLE
Patch 1: Updates

### LINKED ISSUE #184: Bug: Prettier is failing
**Describe the bug**

The GH Action Prettier is failing on every PR. Please fix this
**To Reproduce**
Not reproduceable.

**Expected behavior**
It's supposed to work

**Browser Information**

ChromeOS Version 91.0.4472.81 (Official Build) (64-bit)
**Additional context**

NONE

### PR DESCRIPTION
**Pull request by imlens**
**Resolves**

Resolves #184 

**Changes**

I've added some files. There is a bug, but I will fix that. Do not merge.
**Reason for changes**

Prettier is breaking. so that is one
**Tests**

Tested locally.

### CODE REVIEW

**Comment by FunctionalMetatable** [comment]
I'm sure this doesn't change **anything** related to prettier..

**Comment by NotDiamondZ** [comment]
It does. Just merge when I'm ready.

Sent with [ProtonMail](https://protonmail.com/) Secure Email.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐

**Comment by NotDiamondZ** [comment]
There was a conflict, I resolved it. I removed the bug that was making Prettier fail.

Sent with [ProtonMail](https://protonmail.com/) Secure Email.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐

**Comment by xItzDiamondx** [comment]
Ima merge

**Comment by xItzDiamondx** [comment]
If Shluffy approves

File: 404.html [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <link rel="icon" type="assets/image/png" href="https://scratch-bookmarklets.github.io/assets/img/scratchbookmarklets.png" />
+  <base href="https://scratch-bookmarklets.github.io" />
+  <link rel="stylesheet" href="assets/css/style.css" />
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>404 - Scratch Bookmarklets</title>
```
Comment (Shluffy) [inline 404.html:11]: thoughts on this anybody?

File: assets/json/package.json [hunk 2]
```diff
@@ -12,6 +12,8 @@
     "scratch",
     "shluffy",
     "xitzdiamondx",
+    "notdiamondz",
+    "brourbeinsus",
```
Comment (Shluffy) [inline assets/json/package.json:16]: pls use only 1

File: bookmarklets.html [hunk 3]
```diff
@@ -54,6 +54,13 @@ <h4>Scratch Bookmarklets</h4>
           }
         });
     </script>
-    <script src="assets/js/script.js"
```
Comment (Shluffy) [inline bookmarklets.html:57]: this? removed here but not on other pages?

File: README.md [hunk 4]
```diff
@@ -1,14 +1,12 @@
 # Scratch-Bookmarklets.github.io
 
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
```
Comment (Shluffy) [inline README.md:3]: maybe don't remove

Comment (Shluffy) [reply README.md:3]: or change

File: README.md [hunk 5]
```diff
@@ -1,14 +1,12 @@
...
 <!-- ALL-CONTRIBUTORS-BADGE:END -->
-
-The site that contains a place to copy the bookmarklets code from, as well as more about the team.
-
+<br/><br/>
 <img src="https://user-images.githubusercontent.com/81493936/118314988-839db680-b4ba-11eb-970d-1a9d380a10af.png" width="1000">
-
-## See the LIVE site at [scratch-bookmarklets.github.io](https://scratch-bookmarklets.github.io)
+<h1 align="center">Scratch Bookmarklets</h1>
+<h2>See the site LIVE at our website; <a href="https://bit.ly/ScratchBookmarklets" title="Website">Scratch Bookmarklets</a></h2>
```
Comment (Shluffy) [inline README.md:9]: remove live pls

**Review by Shluffy — CHANGES_REQUESTED** [review]

File: assets/json/package.json (hunk 2 above)
Comment (NotDiamondZ) [reply assets/json/package.json:16]: ok

File: 404.html (hunk 1 above)
Comment (NotDiamondZ) [reply 404.html:11]: that is so it works with microsoft edge and ie. it is automated in vs code

**Pull request closed without merging**